analyzer/runtime/incoming.go

```diff
@@ -246,6 +248,7 @@ func extractRound(b *block.Block, txrs []*sdkClient.TransactionWithResults, logg
 		}
 		blockTransactionData.Raw = cbor.Marshal(txr.Tx)
 		blockTransactionData.RawResult = cbor.Marshal(txr.Result)
+		blockTransactionData.Size = len(blockTransactionData.Raw)
```

Comment:
can you replace `txSize` below (~line 343) with this and move the comment up here

Reply:
Sure, updated